docs/src/SUMMARY.md

```diff
@@ -7,6 +7,7 @@
 # Project setup
 - [Building](./build.md)
 - [Testing](./test.md)
+- [Verification](./verification.md)
```

Comment:
imo this should be in the dev notes section, something like `Verification in tests`